x-pack/plugins/timelines/public/components/t_grid/toolbar/bulk_actions/alert_status_bulk_actions.tsx

```diff
@@ -145,8 +149,11 @@ export const AlertStatusBulkActionsComponent = React.memo<StatefulAlertStatusBul
     );
 
     const statusBulkActionItems = useStatusBulkActionItems({
-      currentStatus: filterStatus,
+      indexName,
       eventIds: Object.keys(selectedEventIds),
+      currentStatus: filterStatus,
+      ...(showClearSelection ? { query } : {}),
+      // query,
```

Comment:
nit: What's this commented out thing here? We should remove commented out things.